src/types/form.ts

```diff
@@ -248,6 +248,10 @@ export type IPopulatedEmailForm = IPopulatedForm & IEmailForm
 
 export interface IFormModel extends Model<IFormSchema> {
   getOtpData(formId: string): Promise<FormOtpData | null>
+  deleteFormLogic(
+    formId: IPopulatedForm,
```

Comment:
we should use consistent naming - if we're passing the full form then call the parameter `form`, but if we can afford to just pass the form ID then we should stick to `formId`

Reply:
edited